src/app/routes/stake/stake-list/stake-list-card/stake-list-card.component.scss

```diff
@@ -0,0 +1,116 @@
+::ng-deep .mat-card-header-text {
+    margin: 0px !important;
+    display: none; 
+}
+
+*{
+    font-family: 'Rajdhani', sans-serif;
+}
+
+.text-subtitle{
+    color: #A6A5AC;
+    font-size: .7rem;
+}
+
+.mat-card{
+    background: #242230;
+    padding: 0;
+
+    > * {
+        padding: 1rem;
+    }
+}
+
+.card-header{
+    display: flex;
+    justify-content: space-between;
+    background-color: inherit;
+    padding-bottom: 0 !important;
+    border: 0px;
+
+    .card-header-image{
+        width: 2.8rem;
+        height: 2.9rem;
+    }
+
+    .mat-card-title{
+        font-size: 1rem;
+        margin-bottom: .5rem;
+    }
+}
+
+.mat-card-content{
+
+    p{
```

Comment:
I prefer to use classes instead of styling through html tags. What do you think about it?